tests/FSharp.Data.Tests/HtmlParser.fs

```diff
@@ -96,6 +96,18 @@ let ``Can handle attributes with no value``() =
         ]
     node.Attributes() |> should equal expected
 
+[<TestCase("var r =\"</script>\"")>]
+[<TestCase("var r ='</script>'")>]
+[<TestCase("var r =/</g")>]
+[<TestCase("//</script>\n")>]
```

Comment:
Can you add the opposite case, not having the \n at the end to make sure it fails